specs/experimental/cannon-fault-proof-vm-mt.md

```diff
@@ -0,0 +1,557 @@
+| 4222 | gettid        | 🚫              | 🚫               | 🚫           | 🚫               | Returns the active thread's threadID field.                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| 4238 | futex         | uint32 addr     | uint32 futex_op  | uint32 val   | uint32 \*timeout | Supports `futex_op`'s `FUTEX_WAIT_PRIVATE` (128) and `FUTEX_WAKE_PRIVATE` (129). Other operations set errno to `0x16`.                                                                                                                                                                                                                                                                                                                                                                              |
+| 4005 | open          | 🚫              | 🚫               | 🚫           | 🚫               | Sets errno to `0x9`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| 4166 | nanosleep     | 🚫              | 🚫               | 🚫           | 🚫               | Preempts the active thread and returns 0.                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| 4263 | clock_gettime | uint32 clock_id | uint32 addr      | 🚫           | 🚫               | Supports `clock_id`'s `REALTIME`(0) and `MONOTONIC`(1). For other `clock_id`'s, sets errno to `0x16`.  Calculates a deterministic time value based on the state's `step` field and a constant `HZ` (10,000,000) where `HZ` represents the approximate clock rate (steps / second) of the FPVM: $$\text{seconds} = \text{step} / \text{HZ}$$ $$ \text{nsecs} = \frac{(\text{step} \% \text{HZ}) \times 10^9}{\text{HZ}} $$ Seconds are set at memory address `addr` and nsecs are set at `addr + 4`. |
```

Comment:
The LaTeX math expression isn't displayed correctly here. Try using `$...$` instead.

Reply:
How are you previewing it?  I think it looks ok on my machine (running `just serve`):
<img width="910" alt="Screenshot 2024-09-30 at 4 51 34 PM" src="https://github.com/user-attachments/assets/6716a49c-a084-4890-b3bc-5d646a40a775">

Reply:
Ah. Maybe it only works on the webapp. I'm previewing it on github - https://github.com/ethereum-optimism/specs/blob/cd6ef8e7d96cfba604daab621bbddddb72220e77/specs/experimental/cannon-fault-proof-vm-mt.md#supported-syscalls

Reply:
We can merge it if it only works on specs.optimism.io. But worth fixing it on gh too.

Reply:
I couldn't get alternative LaTeX expressions rendering correctly.  Switched to useing a `code` style - wdyt?:

<img width="967" alt="Screenshot 2024-10-01 at 3 00 39 PM" src="https://github.com/user-attachments/assets/7f2f3a4c-503e-40c9-89c7-3198d17befd5">

Reply:
LGTM. Thanks!